src/test/ui/unique/unique-containing-tag.rs

```diff
@@ -19,9 +17,9 @@ pub fn main() {
     }*/
 
     /*alt x {
-      box t1(a) {
+      Box::new(t1(a) {
         assert_eq!(a, 10);
-      }
+      })
       _ { panic!(); }
     }*/
 }
```

Comment:
The comments in this test contain ancient match syntax.

Reply:
Yeah the PR touched a bunch of extremely ancient tests, observable on things like different code styles or such.